pkg/config/config.go

```diff
@@ -47,13 +33,13 @@ func (dc *DaemonConfig) ValidateConfig() error {
 		return fmt.Errorf("ValidateConfig(): invalid \"PeriodicUpdate\" value %v", dc.PeriodicUpdate)
 	}
 
-	if dc.SubnetManager.Plugin == "" {
+	if dc.Plugin == "" {
 		return fmt.Errorf("ValidateConfig(): no plugin selected, supported plugins %v", supportedPlugins)
 	}
 
 	if !dc.isSupportedPlugin() {
```

Comment:
A general comment: if vendors provide their own plugin do we need to enforce if the plugin is supported?

Reply:
yes you are right I will remove it